src/client/components/forms/relationship.js

```diff
@@ -102,7 +129,15 @@ class RelationshipEditor extends React.Component {
 					dataHelper.getEntityLink(this.props.entity);
 			});
 	}
-
+	/**
+	* This function goes through the list of updated relationships
```

Comment:
Gets a list of relationships on the form from getValue(), and adds additional information derived from the state that is used internally